Havtorn/Source/Engine/Core/MathTypes/Matrix.h

```diff
@@ -235,10 +239,11 @@ namespace Havtorn
 		rotationMatrix.OrthoNormalize();
 		
 		SVector euler;
+		// OrhothoNormalized rotation matrix is not used!
```

Comment:
Should be, please replace M with rotationMatrix

Reply:
Did a test on another branch (Math), using _RotationMatrix_ vs using _this_ made no difference in the Euler values. 
Could be something to consider in the future.